docs/status.md

```diff
@@ -57,7 +57,7 @@ one of the following tracking labels.
 | broadcast                | no            | yes*         | yes*           | yes             | no          |
 | broadcast_in_dim         | yes           | yes          | infeasible     | yes             | no          |
 | case                     | yes           | revisit      | yes            | no              | no          |
-| cbrt                     | no            | yes*         | yes*           | yes             | no          |
+| cbrt                     | yes           | yes          | yes            | yes             | no          |
```

Comment:
Does is fail for complex types on during compile time

Reply:
If so open a ticket

Reply:
Here, the input and output type is explicitly `HLO_FpTensor`, so it will fail during compile time.
https://github.com/openxla/stablehlo/blob/e9cacdccbb73b8b03b4de6e8be110eec94dd3d31/stablehlo/dialect/StablehloOps.td#L218-L219

Reply:
[shape_inference.cc](https://github.com/tensorflow/tensorflow/blob/49959d2f7d5e497edf90d6cdebc9afa3247234b7/tensorflow/compiler/xla/service/shape_inference.cc#L265-L274) allows both floating-point and complex inputs/outputs. Similar ops - `rsqrt` and `sqrt` - also allow complex inputs/outputs. It looks like this is a bug in `cbrt`, and we should have a ticket to allow complex tensors as well? (And Verification should be "no"?).

Reply:
Thanks, I've created an issue #587 to track this.